README.md
# Danta

changelog.txt
Version: 4.6.0
  - Updated Bootstrap to version 5.1.2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.5.0
  - Fixed slider issue in testimonials and portfolio details sections

Version: 4.4.0
  - Updated Bootstrap to version 5.1.1
  - Updated all outdated third party vendor libraries to their latest versions
  - Improved and updated dev version gulp scripts

Version: 4.3.0
  - Updated Bootstrap to version 5.0.1
  - Updated all outdated third party vendor libraries to their latest versions
  - Fixed navigation links focus color

Version: 4.2.0
  - Updated Bootstrap to version 5.0.0 Final
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.1.0
  - Updated Bootstrap to version 5.0.0-beta3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to V3.1

Version: 4.0.1
  - Updated Bootstrap to version 5.0.0-beta2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.0.0
  - The template does not require jQuery anymore
  - Removed jQuery and all the jQuery plugins
  - The assets/js/main.js was rewritten completely with vanilla Javascript. No more jQuery code
  - Restructured the dev version for better development experience
  - Updated the PHP Email Form to V3.0 - No jQuery dependency. Added attachment support

Version: 3.0.0
  - Initial release with Bootstrap v5.0 Beta 1

Version: 2.3.0
  - Updated Bootstrap to version 4.5.3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to v2.3
  - Other small fixes and improvements

Version: 2.2.0
  - Added smooth scroll on page load with hash links in the url
  - Other small fixes and improvements
  
Version: 2.1.0
  - Updated Bootstrap to version 4.5.0
  - Updated the PHP Email Form library to version 2.0 with reCaptcha support
  - Added portfolio-details.html template
  - Aded inner-page.html tempalte
  - Added navigation active state on scroll
  - Updated all the third party vendor libraries to their latest versions
  - Other small fixes and improvements
  
Version: 2.0.1
  - Fixed animation on scroll effect not applying after portfolio filters clicked

Version: 2.0.0
  - The template was rebuilt from scratch with the latest Bootstrap version (4.4.1)
  - Added SMPTP support for the contact form script (Pro)
  - Added NodeJS NPM Development version (Pro unlimited & Membership members)
  
Version: 1.0.0
- Initial Release


Sobre_Nosotros.php

      

<!DOCTYPE html>
<html lang="es">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-FYPX81JMET"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-FYPX81JMET');
</script>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Dantaventura</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.ico" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: Squadfree - v4.6.0
  * Template URL: https://bootstrapmade.com/squadfree-free-bootstrap-template-creative/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body id="sobre">
  
<header id="header" class="fixed-top ">
    <div class="container d-flex align-items-center justify-content-between">

      <div class="logo">
        <h1 class="text-light"><a href="index.html"><span>Dantaventura</span></a></h1>
        <!-- Uncomment below if you prefer to use an image logo -->
        <!-- <a href="index.html"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->
      </div>

      <nav id="navbar" class="navbar">
        <ul>
          <li><a class="nav-link scrollto active" href="#hero">Inicio</a></li>
          <li><a class="nav-link scrollto" href="#about">Quiénes Somos</a></li>
          <li><a class="nav-link scrollto" href="#portfolio">Galeria de Imágenes</a></li>
         <li class="dropdown"><a href="#services"><span>Servicios</span> <i class="bi bi-chevron-down"></i></a>
            <ul>
              <li><a href="#">Alojamientos</a></li>
             <li><a href="#">Esparcimiento</a></li>
              <li><a href="#">Comercio</a></li>
              <li><a href="#">Otros</a></li>
            </ul>
          </li>
          <li><a class="nav-link scrollto" href="#contact">Contáctenos</a></li>
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->

<div class=" row sobreNosotros container-fluid">
<div class="col-md-6">
<img src="./assets/img/logodanta.png" alt="">
</div>
<div class="col-md-6">
<h1>Quienes Somos</h1>
</div>
 
<table class="table table-hover">
  <tr>
    <th>Categoría</th>
    <th>Modificar</th>
  </tr>
  <tr>
    <td>Sobre la Danta</td>
    <td><button type="button" class="btn"><i class="bi bi-pencil-square"></i></button></td>
  </tr>
  <tr>
    <td>Historia</td>
    <td><button type="button" class="btn"><i class="bi bi-pencil-square"></i></button></td>
  </tr>
  <tr>
   <td>Donde queda ubicada</td>
    <td><button type="button" class="btn"><i class="bi bi-pencil-square"></i></button></td>
  </tr>
  <tr>
    <td>Como llegar</td>
    <td><button type="button" class="btn"><i class="bi bi-pencil-square"></i></button></td>
  </tr>
  <tr>
    <td>Atractivos turísticos</td>
    <td><button type="button" class="btn"><i class="bi bi-pencil-square"></i></button></td>
  </tr>
</table>
</div>

    <!--<div class="col-md-6 d-flex justify-content-center bg-secondary mb-3 Primero ">
      <form action="" method="POST">
          <table class="table table-hover"> 
              <thead>
                  <tr>
                      <th>Categoria</th>
                      <th>Modificar</th>
                  </tr>
              </thead>
                <tbody>
                  <tr><td>Sobre la Danta          </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Historia                </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Donde queda Ubicada     </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Como Llegar             </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Atractivos Turisticos   </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                </tbody>
          </table>
      </form>
    </div>-->
</body>
</html>

modificaciones.php
<!DOCTYPE html>
<html lang="es">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-FYPX81JMET"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-FYPX81JMET');
</script>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Dantaventura</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.ico" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: Squadfree - v4.6.0
  * Template URL: https://bootstrapmade.com/squadfree-free-bootstrap-template-creative/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body>
    <div class="col-md-12 d-inline-flex  mb-3 Primero ">
      <form action="" method="POST">
        <table class="table table-hover"> 
            <thead>
                <tr>
                    <th>Categoria</th>
                </tr>
            </thead>

            <tbody>
                <tr><th>Modificar Informacion Sobre La Danta              </th><td><input type="text" name="danta" class="form-control mt-6" placeholder="Sobre la Danta">          </tr></td>
                <tr><th>Modificar Informacion sobre La Historia           </th><td><input type="text" name="historia" class="form-control mt-6" placeholder="Historia" >               </tr></td>
                <tr><th>Modificar Informacion Sobre Donde Queda Ubicada   </th><td><input type="text" name="quien" class="form-control mt-6" placeholder="Donde queda Ubicada">     </tr></td>
                <tr><th>Modificar Informacion Sobre Como LLegar           </th><td><input type="text" name="como" class="form-control mt-6" placeholder="Como Llegar" >            </tr></td>
                <tr><th>Modificar Informacion Sobre Atractivos Turisticos </th><td><input type="text" name="atractivo" class="form-control mt-6" placeholder="Atractivos Turisticos">   </tr></td>
            </tbody>
        </table>
      </form>
    </div>
    <?php
      @$danta    = $_POST["danta"];
      @$historia = $_POST["historia"];
      @$quien    = $_POST["quien"];
      @$como     = $_POST["como"];
      @$atrac    = $_POST["atractivo"];
      @$tabla = "categoria";
      @$datos = array("danta"=>$_POST["danta"],
              "historia"=>$_POST["historia"],
              "quien"=>$_POST["quien"],
              "como"=>$_POST["como"],
              "atractivo"=>$_POST["atractivo"]);
    ?>
</body>
</html>

form_Sobre_Nosotros.php
<!DOCTYPE html>
<html lang="es">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-FYPX81JMET"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-FYPX81JMET');
</script>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Dantaventura</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.ico" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: Squadfree - v4.6.0
  * Template URL: https://bootstrapmade.com/squadfree-free-bootstrap-template-creative/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body id="sobre">
  
<header id="header" class="fixed-top ">
    <div class="container d-flex align-items-center justify-content-between">

      <div class="logo">
        <h1 class="text-light"><a href="index.html"><span>Dantaventura</span></a></h1>
        <!-- Uncomment below if you prefer to use an image logo -->
        <!-- <a href="index.html"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->
      </div>

      <nav id="navbar" class="navbar">
        <ul>
          <li><a class="nav-link scrollto active" href="#hero">Inicio</a></li>
          <li><a class="nav-link scrollto" href="#about">Quiénes Somos</a></li>
          <li><a class="nav-link scrollto" href="#portfolio">Galeria de Imágenes</a></li>
         <li class="dropdown"><a href="#services"><span>Servicios</span> <i class="bi bi-chevron-down"></i></a>
            <ul>
              <li><a href="#">Alojamientos</a></li>
             <li><a href="#">Esparcimiento</a></li>
              <li><a href="#">Comercio</a></li>
              <li><a href="#">Otros</a></li>
            </ul>
          </li>
          <li><a class="nav-link scrollto" href="#contact">Contáctenos</a></li>
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->
  <div class=" row sobreNosotros container-fluid">
<div class="col-md-6">
<img src="./assets/img/logodanta.png" alt="">
</div>
<div class="col-md-6">
<h1>Titulo de contenido a Modificar</h1>
</div>


    <form action="">
        <textarea class="form-control" name="" id="" cols="30" rows="10"></textarea>
              <input type="submit" value="Modificar" class="btn">
    </form>


    <!--<div class="col-md-6 d-flex justify-content-center bg-secondary mb-3 Primero ">
      <form action="" method="POST">
          <table class="table table-hover"> 
              <thead>
                  <tr>
                      <th>Categoria</th>
                      <th>Modificar</th>
                  </tr>
              </thead>
                <tbody>
                  <tr><td>Sobre la Danta          </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Historia                </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Donde queda Ubicada     </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Como Llegar             </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                  <tr><td>Atractivos Turisticos   </td>
                  <td><button type="button" class="btn btn-success"><i class="bi bi-pencil-square"></i></button></td></tr>
                </tbody>
          </table>
      </form>
    </div>-->
</body>
</html>

assets/js/main.js
/**
* Template Name: Squadfree - v4.6.0
* Template URL: https://bootstrapmade.com/squadfree-free-bootstrap-template-creative/
* Author: BootstrapMade.com
* License: https://bootstrapmade.com/license/
*/
(function() {
  "use strict";

  /**
   * Easy selector helper function
   */
  const select = (el, all = false) => {
    el = el.trim()
    if (all) {
      return [...document.querySelectorAll(el)]
    } else {
      return document.querySelector(el)
    }
  }

  /**
   * Easy event listener function
   */
  const on = (type, el, listener, all = false) => {
    let selectEl = select(el, all)
    if (selectEl) {
      if (all) {
        selectEl.forEach(e => e.addEventListener(type, listener))
      } else {
        selectEl.addEventListener(type, listener)
      }
    }
  }

  /**
   * Easy on scroll event listener 
   */
  const onscroll = (el, listener) => {
    el.addEventListener('scroll', listener)
  }

  /**
   * Navbar links active state on scroll
   */
  let navbarlinks = select('#navbar .scrollto', true)
  const navbarlinksActive = () => {
    let position = window.scrollY + 200
    navbarlinks.forEach(navbarlink => {
      if (!navbarlink.hash) return
      let section = select(navbarlink.hash)
      if (!section) return
      if (position >= section.offsetTop && position <= (section.offsetTop + section.offsetHeight)) {
        navbarlink.classList.add('active')
      } else {
        navbarlink.classList.remove('active')
      }
    })
  }
  window.addEventListener('load', navbarlinksActive)
  onscroll(document, navbarlinksActive)

  /**
   * Scrolls to an element with header offset
   */
  const scrollto = (el) => {
    let header = select('#header')
    let offset = header.offsetHeight

    if (!header.classList.contains('header-scrolled')) {
      offset -= 24
    }

    let elementPos = select(el).offsetTop
    window.scrollTo({
      top: elementPos - offset,
      behavior: 'smooth'
    })
  }

  /**
   * Toggle .header-scrolled class to #header when page is scrolled
   */
  let selectHeader = select('#header')
  if (selectHeader) {
    const headerScrolled = () => {
      if (window.scrollY > 100) {
        selectHeader.classList.add('header-scrolled')
      } else {
        selectHeader.classList.remove('header-scrolled')
      }
    }
    window.addEventListener('load', headerScrolled)
    onscroll(document, headerScrolled)
  }

  /**
   * Back to top button
   */
  let backtotop = select('.back-to-top')
  if (backtotop) {
    const toggleBacktotop = () => {
      if (window.scrollY > 100) {
        backtotop.classList.add('active')
      } else {
        backtotop.classList.remove('active')
      }
    }
    window.addEventListener('load', toggleBacktotop)
    onscroll(document, toggleBacktotop)
  }

  /**
   * Mobile nav toggle
   */
  on('click', '.mobile-nav-toggle', function(e) {
    select('#navbar').classList.toggle('navbar-mobile')
    this.classList.toggle('bi-list')
    this.classList.toggle('bi-x')
  })

  /**
   * Mobile nav dropdowns activate
   */
  on('click', '.navbar .dropdown > a', function(e) {
    if (select('#navbar').classList.contains('navbar-mobile')) {
      e.preventDefault()
      this.nextElementSibling.classList.toggle('dropdown-active')
    }
  }, true)

  /**
   * Scrool with ofset on links with a class name .scrollto
   */
  on('click', '.scrollto', function(e) {
    if (select(this.hash)) {
      e.preventDefault()

      let navbar = select('#navbar')
      if (navbar.classList.contains('navbar-mobile')) {
        navbar.classList.remove('navbar-mobile')
        let navbarToggle = select('.mobile-nav-toggle')
        navbarToggle.classList.toggle('bi-list')
        navbarToggle.classList.toggle('bi-x')
      }
      scrollto(this.hash)
    }
  }, true)

  /**
   * Scroll with ofset on page load with hash links in the url
   */
  window.addEventListener('load', () => {
    if (window.location.hash) {
      if (select(window.location.hash)) {
        scrollto(window.location.hash)
      }
    }
  });

  /**
   * Porfolio isotope and filter
   */
  window.addEventListener('load', () => {
    let portfolioContainer = select('.portfolio-container');
    if (portfolioContainer) {
      let portfolioIsotope = new Isotope(portfolioContainer, {
        itemSelector: '.portfolio-item',
        layoutMode: 'fitRows'
      });

      let portfolioFilters = select('#portfolio-flters li', true);

      on('click', '#portfolio-flters li', function(e) {
        e.preventDefault();
        portfolioFilters.forEach(function(el) {
          el.classList.remove('filter-active');
        });
        this.classList.add('filter-active');

        portfolioIsotope.arrange({
          filter: this.getAttribute('data-filter')
        });
        portfolioIsotope.on('arrangeComplete', function() {
          AOS.refresh()
        });
      }, true);
    }

  });

  /**
   * Initiate portfolio lightbox 
   */
  const portfolioLightbox = GLightbox({
    selector: '.portfolio-lightbox'
  });

  /**
   * Portfolio details slider
   */
  new Swiper('.portfolio-details-slider', {
    speed: 400,
    loop: true,
    autoplay: {
      delay: 5000,
      disableOnInteraction: false
    },
    pagination: {
      el: '.swiper-pagination',
      type: 'bullets',
      clickable: true
    }
  });

  /**
   * Testimonials slider
   */
  new Swiper('.testimonials-slider', {
    speed: 600,
    loop: true,
    autoplay: {
      delay: 5000,
      disableOnInteraction: false
    },
    slidesPerView: 'auto',
    pagination: {
      el: '.swiper-pagination',
      type: 'bullets',
      clickable: true
    },
    breakpoints: {
      320: {
        slidesPerView: 1,
        spaceBetween: 40
      },

      1200: {
        slidesPerView: 3,
      }
    }
  });

  /**
   * Animation on scroll
   */
  window.addEventListener('load', () => {
    AOS.init({
      duration: 1000,
      easing: "ease-in-out",
      once: true,
      mirror: false
    });
  });

})()